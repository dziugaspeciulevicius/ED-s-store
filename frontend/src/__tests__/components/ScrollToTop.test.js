import React from "react";
import { shallow } from "enzyme";
import "../../setupTests";
import ScrollToTop from "../../components/ScrollToTop.component";
jest.mock("../../components/ScrollToTop.component");

describe("ScrollToTop tests", () => {
  it("Renders <ScrollToTop /> component correctly", () => {
    // shallow creates instance of a component
    expect(shallow(<ScrollToTop />)).toMatchSnapshot();
  });
});
