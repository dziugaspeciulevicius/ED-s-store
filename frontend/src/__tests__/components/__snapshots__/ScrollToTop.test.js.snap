// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ScrollToTop tests Renders <ScrollToTop /> component correctly 1`] = `ShallowWrapper {}`;
